.ci/install_crio.sh

```diff
@@ -55,23 +55,21 @@ then
 		else
 			crio_version=$(get_version "externals.crio.meta.openshift")
 		fi
+		crictl_version=$(get_version "externals.crio.meta.crictl")
+		crictl_tag_prefix=""
 	else
 		crio_version=$(get_version "externals.crio.version")
+		crictl_version=$(get_test_version "externals.critools.version")
+		crictl_tag_prefix="v"
 	fi
 
 	git fetch
 	git checkout "${crio_version}"
 fi
 
-echo "Get CRI Tools"
-critools_repo="${kubernetes_sigs_org}/cri-tools"
-go get "$critools_repo" || true
-pushd "${GOPATH}/src/${critools_repo}"
-critools_version=$(grep "ENV CRICTL_COMMIT" "${GOPATH}/src/${crio_repo}/Dockerfile" | cut -d " " -f3)
-git checkout "${critools_version}"
-go install ./cmd/crictl
-sudo install "${GOPATH}/bin/crictl" /usr/bin
-popd
+echo "Installing CRI Tools"
+crictl_url=https://github.com/kubernetes-sigs/cri-tools/releases/download/v$crictl_version/crictl-$crictl_tag_prefix$crictl_version-linux-"$(${cidir}/kata-arch.sh -g)".tar.gz
+wget -Ls "$crictl_url" | tar xfz - -C /usr/local/bin
```

Comment:
I think you might need to re-push as this is still showing as `wget`.

Reply:
Thanks for the hint, you're right. :) Re-pushed my changes.